### ISSUE TITLE
the learning rate of SGD in the NUS-WIDE dateset

### ISSUE THREAD

**Issue by SuperEliGun** [jiangqy/DCMH-CVPR2017#12]
Hello, thanks for the code.
I have a question about the learning rate of SGD in the NUS-WIDE dataset. I am trying to use your code in the NUS-WIDE dataset, but I found that the result is so bad when using the same SGD optimizer and learning rate as the MIRFLICKR-25K dataset.

**Comment by jiangqy** [comment]
Which code do you use? tensorflow or matconvnet? For tensorflow code, I didn't test on NUSWIDE dataset. You might need to tune the learning rate by yourself.

**Comment by iTomxy** [comment]
then what's the learning rate for the matlab version ?

**Comment by jiangqy** [comment]
@iTomxy 
for NUSWIDE:
para.maxIter = 500;
para.lr_txt = logspace(-1.5,-3,para.maxIter);
para.lr_img = logspace(-1.5,-3,para.maxIter);
